### DISCUSSION TITLE
Meetup - Weltvermessen mit OpenDataCam (May 24 2022, Remote or In-Person)

### DISCUSSION THREAD

**Discussion by vsaw** [opendatacam/opendatacam#538] [Show and tell]
There's meetup hosted by @yauh featuring OpenDataCam. Sign up [here](https://www.meetup.com/codecentric-Dortmund-tech-talk/events/285859711/) if you want to attend in Person or join remotely.

Tuesday, May 24, 2022, 6:00 PM to 8:00 PM CEST
[codecentric AG](https://www.google.com/maps/search/?api=1&query=51.51303%2C%207.457149) Hoher Wall 15 · Dortmund

**Comment by yauh** [comment]
Recording was b0rked, but the slides are online. https://www.slideshare.net/stephanhochhaus/weltvermessen-mit-opendatacam-wie-ich-einmal-eine-viertelmillionen-autos-gezhlt-habe
